plc4j/integrations/apache-nifi/README.md

```diff
@@ -48,19 +49,66 @@ This applies to all Plc4x processors:
     ```
     If this JSON is in an attribute `plc4x.addresses` it can be accessed with *Address Text*=`${plc4x.addresses}`. 
 
+
```

Comment:
The response isn't converted into Avro, an Avro _schema_ is produced for the PLC Data types